README.md

```diff
@@ -1,3 +1,36 @@
+# Contribution Guide
+
+## Commit Code
+
+**deutsch**
+
+- Fork erstellen
+- Änderungen auf den fork commiten
+- Merge request machen
+- Pull Request reviewen lassen
+
+
+**english**
+
+- create a fork
+- commit changes to the fork
+- create a merge request
+- let the pull request be reviewed
+
+## Bug Report
+
+**deutsch**
+
+Falls du eine Issue anlegst sollte dieses einen klaren Titel mit einer genauen Beschreibung des enthalten. Du solltest so viele relevante Informationen wie möglich angeben und auch ein Code Beispiel, welches den Fehler demonstriert. Ein Bug Report soll dir und anderen helfen  den Fehler zu reproduzieren und einen Fix zu entwickeln.
```

Comment:
> Falls du eine Issue anlegst sollte dieses einen klaren

"Falls du eine Issue anlegst, sollte ..."

> genauen Beschreibung des enthalten

"des" is not necessary here.

> anderen helfen  den Fehler zu reproduzieren

Two whitespaces between *helfen  den*